packages/dev/materials/src/custom/customMaterial.ts

```diff
@@ -10,6 +10,9 @@ import { Color3, Color4 } from "core/Maths/math.color";
 import type { Nullable } from "core/types";
 import type { SubMesh } from "core/Meshes/subMesh";
 
+import "@babylonjs/core/Shaders/default.vertex";
+import "@babylonjs/core/Shaders/default.fragment";

```

Comment:
Why is this not
```suggestion
import "core/Shaders/default.vertex";
import "core/Shaders/default.fragment";

```

Reply:
Because @RaananW told me so :D

Reply:
Just noticed this comment. this was for the user, not for us in core :-)